src/segments/sitecore.go

```diff
@@ -0,0 +1,96 @@
+package segments
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"strings"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type Sitecore struct {
+	props properties.Properties
+	env   platform.Environment
+
+	EnvironmentName    string
+	IsCloudEnvironment bool
+	EnvironmentIcon    string
+}
+
+type SitecoreConfig struct {
+	Endpoints struct {
+		Default struct {
+			Ref        string `json:"ref"`
+			AllowWrite bool   `json:"allowWrite"`
+			Host       string `json:"host"`
+			Variables  struct {
+			} `json:"variables"`
+		} `json:"default"`
+	} `json:"endpoints"`
+}
+
+const (
+	CloudIcon properties.Property = "cloud_icon"
+	OtherIcon properties.Property = "other_icon"
+)
+
+func (s *Sitecore) Enabled() bool {
+	return s.shouldDisplay()
+}
+
+func (s *Sitecore) Template() string {
+	return " {{ if .PackageManagerIcon }} {{ .PackageManagerIcon }} {{ end }} {{ .EnvironmentName }} "
```

Comment:
this is incorrect, right? there's no `. PackageManagerIcon ` property in this segment.

Reply:
stupid mistake late at night. I took some "inspiration" from the node segment.

Reply:
as of your [other comment](https://github.com/JanDeDobbeleer/oh-my-posh/pull/3789/files#r1190690246), this Icon is even unnecessary, I'll remove it